src/contrib/Serializers.js

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
I think you are missing a subfolder `invenio` inside `contrib`, unless you had other ideas in mind!